buildSrc/build.gradle.kts

```diff
@@ -1,9 +1,10 @@
 import ca.cutterslade.gradle.analyze.AnalyzeDependenciesTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
+@Suppress("DSL_SCOPE_VIOLATION")
```

Comment:
Trodde ikke det var nødvendlig lenger? Skulle ikke være nødvendig med gradle 8.1 eller nyere.

Reply:
Jaggu, kunne berre ta bort. Sånn går det når ein kopierer for ukritisk 🙈